### PR TITLE
[air examples] stop using runtime env deps

### PR DESCRIPTION
**Pull request by aslonnie**
mostly using the packages in the image.

### CODE REVIEW

File: release/ray_release/byod/byod_gptj_test.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (can-anyscale) [inline release/ray_release/byod/byod_gptj_test.sh:1]: need to make this file executable it seems

Comment (aslonnie) [reply release/ray_release/byod/byod_gptj_test.sh:1]: yep; fixed.

**Comment by aslonnie** [comment]
running test right now: https://buildkite.com/ray-project/release/builds/19849

**Comment by aslonnie** [comment]
tests all passing.

**Comment by aslonnie** [comment]
I will patch this to master branch.

**Review by can-anyscale — APPROVED** [review]

**Comment by aslonnie** [comment]
I am also merging this to the release branch

the one to merge into master is here:
https://github.com/ray-project/ray/pull/46759

**Pull request merged**